pkg/crc/api/api.go

```diff
@@ -36,7 +36,7 @@ func createAPIServerWithListener(listener net.Listener, config newConfigFunc, ma
 	return apiServer, nil
 }
 
-func (api Server) Serve() {
+func (api Server) Serve() error {
```

Comment:
Will it ever return the error, since no return statement in the function body?

Reply:
http.Serve() traditionally returns error: failure of `accept` should trigger a failure of `Serve` and the server itself.

Reply:
"should trigger a failure", but this is not the case at the moment, right? Ithink this is what Anjan was pointing out.

Reply:
Yes not the case now, I should open a follow up with that. My plan is to handle both http and the Anjan-protocol at some point :)